tests/test_bot.rs

```diff
@@ -73,7 +81,53 @@ mod tests {
                                        Some(&ParseMode::Html),
                                        None,
                                        None,
+                                       None,
                                        None);
         assert!(message.is_ok());
     }
+
+    #[test]
+    fn test_url_chat_actions() {
+        let (token, chat_id) = setup();
+        let bot_url = [BASE_URL, &token].concat();
+        let bot = Bot::new(bot_url).unwrap();
+
+        let message = bot.send_chat_action(&chat_id, &ChatAction::FindLocation);
+        assert!(message.is_ok());
+    }
+
+    #[test]
+    fn test_url_inline_keyboard() {
+        let (token, chat_id) = setup();
+        let bot_url = [BASE_URL, &token].concat();
+        let bot = Bot::new(bot_url).unwrap();
+
+        let mut total = Vec::<Vec<InlineKeyboardButton>>::new();
```

Comment:
I think that `buttons` would be a better name than `total` for this variable binding.